dev-lang/php/php-8.3.21.ebuild

```diff
@@ -253,13 +253,6 @@ src_prepare() {
 	# be running pre-install, in my opinion. Bug 927461.
 	rm ext/fileinfo/tests/bug78987.phpt || die
 
-	# Bug 935379, not yet fixed upstream but looks harmless (ordering
-	# of keys isn't guaranteed AFAICS):
-	#
-	# - https://github.com/php/php-src/issues/14786
-	#
-	rm ext/dba/tests/dba_gdbm.phpt || die
```

Comment:
Can you omit this please? 8.3.21 is already stable on arm, arm64, and ppc64, and we try to avoid modifying stable ebuilds as a general rule.